plugins/reach/lib/model/EntryHandler.php

```diff
@@ -60,8 +89,31 @@ public function getAbortStatusMessage($status): string
 
 	}
 
+	public static function getTaskObjectId(BaseObject $object)
+	{
+		return $object->getEntryId();
+	}
+  
 	public static function retrieveObject($objectId): BaseObject
 	{
 		return entryPeer::retrieveByPK($objectId);
 	}
+
+	public static function hasRestrainingAdminTag($object, $profileId): bool
+	{
+		$reachRestrainAdminTag = kConf::get("reach_restrain_admin_tag", kConfMapNames::RUNTIME_CONFIG, null);
+		if(in_array($reachRestrainAdminTag, $object->getAdminTagsArr()))
```

Comment:
in_array is doing a loose search ==, so In case reachRestrainAdminTag equals null, if may return **true** if $object->getAdminTagsArr() contains any of the following - 0,0.0, false, '', '0',[], null
So better check if the needle is not null before checking in the unknown haystack